### PR TITLE
Automate CA certificate bundle update

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

**Comment by kesselb** [comment]
Thank you :+1: 

If the update is automated the old check can probably go:

https://github.com/nextcloud/server/blob/master/build/ca-bundle-checker.sh
https://github.com/nextcloud/server/blob/master/autotest-checkers.sh

**Comment by nickvergessen** [comment]
Interesting that we have this check and it never failed....

**Comment by nickvergessen** [comment]
https://github.com/nextcloud/server/blob/3a75e14844ab94a3c9aa08d7db4657610c352de1/build/ca-bundle-checker.sh#L3-L6

uuuuuuuuuuuuuuff

**Review by CarlSchwan — APPROVED** [review]

**Comment by kesselb** [comment]
> Interesting that we have this check and it never failed....

It failed sometimes ;) However the check for a certain branch name was always unstable. The schedule based workflow is much better.

File: .github/workflows/update-cacert-bundle.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
```
Comment (skjnldsv) [inline .github/workflows/update-cacert-bundle.yml:4]: IS it ?

**Review by skjnldsv — APPROVED** [review]

File: .github/workflows/update-cacert-bundle.yml (hunk 1 above)
Comment (nickvergessen) [reply .github/workflows/update-cacert-bundle.yml:4]: Well it's templated on top of the update-christophwurst-nextcloud one, so it's not wrong :P

File: .github/workflows/update-cacert-bundle.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+name: Update CA certificate bundle
```
Comment (CarlSchwan) [inline .github/workflows/update-cacert-bundle.yml:1]: Maybe add your copyright here. I'll be fine if we could adopt spdx notation to make it shorter

```
# SPDX-FileCopyrightText: 2022 Your name <email>
# SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment (nickvergessen) [reply .github/workflows/update-cacert-bundle.yml:1]: I don't think it's "high" enough to have copyright

**Pull request merged**